libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -150,6 +150,16 @@ AudioSegmentSampleView WaveClip::GetSampleView(
       start + TimeToSamples(mTrimLeft), length, mayThrow);
 }
 
+AudioSegmentSampleView WaveClip::GetSampleView(
+   size_t iChannel, double t0, double t1, bool mayThrow) const
+{
+   assert(iChannel < GetWidth());
+   const auto start = TimeToSamples(std::max(0., t0));
```

Comment:
I think this was wrong:  it should be `TimeToSamples(t0 - GetPlayStartTime())`

Reply:
No, the user is already supposed to give clip-relative times.